src/app/components/footer/footer.controller.js

```diff
@@ -1,12 +1,5 @@
 'use strict';
 
 export default class FooterController {
```

Comment:
Please, remove controller if you don't use it.

Reply:
Done